### PR TITLE
Add Hackathon March 2022 Registration Form

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: markdown/events/2022/hackathon-march-2022.md [hunk 1]
```diff
@@ -13,8 +13,11 @@ location_name: Gather town and Slack.
 Please join us for the March 2022 nf-core hackathon!
 This event is entirely virtual (we will be back in [gather.town](https://gather.town/)!) and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:
 
-More details will come soon, registration will open February 2022.
-Please check back here for details and keep an eye out on twitter and on Slack,
+Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/march-2022).
```
Comment (maxulysse) [inline markdown/events/2022/hackathon-march-2022.md:16]:
```suggestion
Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/march-2022).
```

**Comment by mashehu** [comment]
Can you please add `import_typeform: true` and embed the form?

File: markdown/events/2022/hackathon-march-2022.md [hunk 2]
```diff
@@ -7,16 +7,19 @@ start_time: "10:00 CET"
...
 location_name: Gather town and Slack.
+import_typeform: true
 ---
 
 
 Please join us for the March 2022 nf-core hackathon!
 This event is entirely virtual (we will be back in [gather.town](https://gather.town/)!) and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:
 
-Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/march-2022).
+Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/march-2022) or in the widge below.
```
Comment (mashehu) [inline markdown/events/2022/hackathon-march-2022.md:17]:
```suggestion
Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/march-2022) or in the widget below.
```

**Comment by jfy133** [comment]
Deifnitley can @mashehu now I know it existed :grimacing: !

image

**Review by mashehu — APPROVED** [review]
Tested locally and looks good.

**Pull request merged**